lib/y2lan_restart_common.pm

```diff
@@ -149,14 +149,24 @@ sub check_network_status {
     }
     assert_script_run 'cat journal.log';                              # print journal.log
     my $is_empty = script_run('[ -s journal.log ]');
+    # empty journal.log indicating that service was not restarted or reloaded
     if ($is_empty) {
-        assert_equals($expected_status, 'no_restart', "Network restart didn't occur, even though it was expected.");
+        assert_equals($expected_status, 'no_restart', "Network service should have been restarted/reloaded. journalctl cannot find events");
     }
+    # Based on the query's filter it is assumed that the service is restarted otherwise it is reloaded.
     else {
-        assert_equals($expected_status, 'restart', "Network restart occurred, even though it wasn't expected.");
+        my $journal_findings = script_output('cat journal.log');
+        my $query_pattern    = "shutting down|ifdown all|Stopping wicked";
```

Comment:
I see potential issue here once this string and the one we have on L74 start to differ, as else here will happen automatically. What's about extracting this to separate variable or constant?

Reply:
yes that could help and improve the readability. i will do that